modules/control-utility/src/main/java/org/apache/ignite/internal/commandline/dr/subcommands/DrRebuildPartitionLogCommand.java

```diff
@@ -55,7 +56,7 @@ public class DrRebuildPartitionLogCommand extends AbstractCommand<DrRebuildParti
     /** {@inheritDoc} */
     @Override public Object execute(GridClientConfiguration clientCfg, Logger logger) throws Exception {
         try (GridClient client = Command.startClient(clientCfg)) {
-            Optional<GridClientNode> firstNodeOpt = client.compute().nodes().stream().findFirst();
+            Optional<GridClientNode> firstNodeOpt = TopologyUtils.anyConnectableServerNode(client.compute());
```

Comment:
'before' a non-connectable node might be chosen, so the change is not equivalent. Is this a fix?

Reply:
I think this is not a problem as well and here is why: even if the code for finding the first node returns a non-empty Optional, value from this Optional is completely ignored just a few lines later when `null` nodeId is passed to `executeTaskByNameOnNode` method.

`null` there means that implementation will use a node from GridClient connection.

Moreover, even if we passed the node from Optional and that node turned out to be non-connectable, we would get an exception as `executeTaskByNameOnNode` method filters nodes by `connectable` flag internally.